src/plugins/gc-checklist/gc-checklist.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @title WET-BOEW Checklist plugin
+ * @overview Plugin contained to show an example of a custom WET checklist plugin
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @delisma
+ */
+( function( $, window, wb ) {
+"use strict";
+
+/*
+ * Variable and function definitions.
+ * These are global to the plugin - meaning that they will be initialized once per page,
+ * not once per instance of plugin on the page. So, this is a good place to define
+ * variables that are common to all instances of the plugin on a page.
+ */
+var componentName = "gc-checklist",
+	selector = ".provisional." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+
+	/**
+	 * @method init
+	 * @param {jQuery Event} event Event that triggered the function call
+	 */
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector ),
+			$elm;
+
+		if ( elm ) {
+			$elm = $( elm );
+
+			var $elms = $elm.find( $( "input[type='checkbox']" ) );
+
+			for ( var i = 0; i < $elms.length; i++ ) {
+				var domNode = $elms.get( i );
+				domNode.tabIndex = 0;
+			}
+			wb.ready( $elm, componentName );
+		}
+	};
+
+// Add your plugin event handler
+$document.on( "click", selector, function( event ) {
+	var elm = $( event.currentTarget ),
+		$elm = $( elm ),
+		data = JSON.parse( $elm.attr( "data-" + componentName ) );
+
+	if ( data ) {
+		if ( $elm.find( "[required]" ).length === $elm.find( "[required]" ).filter( ":checked" ).length ) {
+			$elm.find( data.feedbackTarget ).text( data.feedbackSuccess );
+			$elm.find( data.feedbackTarget ).addClass( "label-success" ).removeClass( "label-danger" );
+		} else {
+			$elm.find( data.feedbackTarget ).text( data.feedbackFailure );
+			$elm.find( data.feedbackTarget  ).addClass( "label-danger" ).removeClass( "label-success" );
+		};
```

Comment:
Do variable caching to avoid to call multiple time the same function. This will improve the code readability and be more optimal.

Like:

```
var $feedbackTarget = $elm.find( data.feedbackTarget );
...
			$feedbackTarget.text( data.feedbackSuccess );
			$feedbackTarget.addClass( "label-success" ).removeClass( "label-danger" );
...
```